core/src/test/scala/com/nvidia/spark/rapids/tool/util/ToolUtilsSuite.scala

```diff
@@ -199,6 +201,17 @@ class ToolUtilsSuite extends FunSuite with Logging {
     result.value.get shouldBe (59 * 1000 + 200)
   }
 
+  /**
+   * Reads the content of a file as UTF-8 and closes the resources.
+   * @param filePath Path to the file
+   * @return Content of the file as a string
+   */
+  private def readFileContentAsUtf8(filePath: String): String = {
+    Using.resource(Source.fromFile(filePath)(StandardCharsets.UTF_8)) {
```

Comment:
I thought that `using` is not available for Scala < 2.13?

Reply:
Yes. Fixed closing of resources.